src/web-ui/src/app/app.component.ts

```diff
@@ -1,8 +1,8 @@
-import { Component, OnDestroy, OnInit } from '@angular/core';
-import { Subscription } from 'rxjs';
-import { UserService } from '../services/user.service';
-import { User } from '../models/user.model';
-import { LanguageService } from '../services/language.service';
+import {Component, OnDestroy, OnInit} from '@angular/core';
+import {Subscription} from 'rxjs';
+import {User} from '../models/user.model';
+import {LanguageService} from '../services/language.service';
+import {AuthenticationService} from "../services/authentication.service";
```

Comment:
A co to? Raz podwójne apostrofy a raz pojedyncze 🤪

Reply:
Świruje mi eslint i prettier, muszę to jeszcze pokonfigurowac w wolnej chwili